lua/telescope/actions.lua

```diff
@@ -12,6 +12,15 @@ local actions = setmetatable({}, {
   end
 })
 
+actions.action_mt = {
+  __add = function (lhs, rhs)
+    return { fn = function(prompt_bufnr)
+      lhs.fn(prompt_bufnr)
+      rhs.fn(prompt_bufnr)
+    end }
```

Comment:
You should return a new table, that uses the metamethod `__call`, so you don't have to check for `fn`.

Are you familiar with `__call`?

Reply:
Was not familiar with it but now i am. Thanks for the good hint.